Planner/src/Planner.java

```diff
@@ -171,25 +186,314 @@ public void actionPerformed(ActionEvent arg0) {
 		tasksList.addElement(task6);
 		tasksList.addElement(task7);
 		
-		Border border = BorderFactory.createLineBorder(Color.BLACK);
+		Border border = BorderFactory.createEtchedBorder(EtchedBorder.RAISED);
 		userLabel.setBorder(border); 
 		taskLabel.setBorder(border);
 		
+		// the panel with list on the left side
 		listPanel.add(userLabel); 
 		listPanel.add(listWithUsers);
 		listPanel.add(Box.createRigidArea(new Dimension(0,20)));
 		listPanel.add(taskLabel);
 		listPanel.add(listWithTasks);
 		listPanel.setBackground(Color.white);
 		listPanel.setLayout(new BoxLayout(listPanel, BoxLayout.Y_AXIS));
-		listPanel.setAlignmentY(LEFT_ALIGNMENT); // correct alignment to the left (labels!) 
 		
-		// layout
+		// the panel with buttons on the top
 		topPanel.setLayout(new BorderLayout());
 		topPanel.setBackground(Color.white);
 		topPanel.add(buttonPanel, BorderLayout.LINE_END);
 		topPanel.add(selectTaskButton, BorderLayout.LINE_START);
 		plannerFrame.add(topPanel, BorderLayout.PAGE_START);
-		plannerFrame.add(listPanel, BorderLayout.LINE_START);
-	}      
+		plannerFrame.add(listPanel, BorderLayout.LINE_START); 
+		
+		// show a JFrame for edit user, when we click on the user in User's list
+		listWithUsers.addListSelectionListener(new ListSelectionListener() {
+			public void valueChanged(ListSelectionEvent evtU) {
+				//JFrame to edit User
+				JFrame editUserFrame = new JFrame("Edit the User");
+				editUserFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE); //clicking on X - close the Planner
```

Comment:
Change this to JFrame.DISPOSE_ON_CLOSE because otherwise the frame with the calendar is being closed as well.